tools/spectrum-scale-driver-snap.sh

```diff
@@ -157,6 +157,11 @@ get_k8snodes_cmd="$cmd describe nodes"
 echo "$get_k8snodes_cmd"
 $get_k8snodes_cmd >> $get_k8snodes 2>&1 || :
 
+
+get_spectum_cmd="$cmd describe ds ${CSI_SPECTRUM_SCALE_LABEL} -n $ns"
+echo "$get_spectum_cmd"
+$get_spectum_cmd >> $get_spectum_cmd 2>&1 || :
```

Comment:
The output needs  to be changed  to an actual target, because it gives an ambiguous redirect when executed:
```
./spectrum-scale-driver-snap.sh: line 165: $get_spectum_cmd: ambiguous redirect
```

Maybe copy the `get_k8snodes` as a template.